src/app/forms/custom-sections/approval-type/approval-type.component.ts

```diff
@@ -30,11 +32,13 @@ export class ApprovalTypeComponent implements OnInit, OnChanges, OnDestroy {
   private destroy$ = new Subject<void>();
   protected chosenApprovalType: string | undefined;
   protected approvalTypeChange = false;
+  protected approvalType: typeof approvalTypeHgvOrPsv | typeof approvalType = approvalType;
   formControls: { [key: string]: FormControl } = {};
 
   constructor(private dfs: DynamicFormService) {}
 
   ngOnInit() {
+    this.approvalType = this.techRecord.techRecord_vehicleType === 'psv' || this.techRecord.techRecord_vehicleType === 'hgv' ? approvalTypeHgvOrPsv : approvalType;
```

Comment:
Note for dev team, in the weights component is some until functions that would have been useful for this 😄 

We should look to make them more available.